quickevent/app/quickevent/plugins/Runs/src/runswidget.cpp

```diff
@@ -220,8 +220,21 @@ void RunsWidget::settleDownInPartWidget(quickevent::gui::PartWidget *part_widget
 		auto *a = new qfw::Action(tr("&Competitors with rented cards"));
 		connect(a, &qfw::Action::triggered, [this]() {
 			qff::MainWindow *fwk = qff::MainWindow::frameWork();
+			quickevent::gui::ReportOptionsDialog dlg(fwk);
```

Comment:
please, bump QE version

Reply:
No problem.
For this PR and PR #956 together or for both independently?

Reply:
for both pls

Reply:
Done